lib/mappings.js

```diff
@@ -23,7 +23,11 @@
 'use strict';
```

Comment:
CNR entry corresponding to this PR is missing?

Reply:
This branch has been reopened.
CNR already is saying:  Update iotagent node library to allow use env variables

Reply:
Are the new changes also related with env vars? If the scope of the adaptation work is wider maybe the message should be adjusted.

Reply:
The changes are related with the update of node iotagent library, that was due to need of env vars.

Reply:
NTC